int64 id                      # 식별 번호
string flower_name            # 꽃 이름
string flower_meaning         # 꽃말
int64 growth_duration_days   # 성장까지 필요한 기간 (일 단위)
int64 watering_cycle          # 물 주는 주기 (일 단위)

int64 growth_state            # 꽃 성장 상태
